.github/workflows/test-env-deploy.yml

```diff
@@ -0,0 +1,63 @@
+name: TEST-ENV-DEPLOYMENT
+
+on:
+  pull_request:
+    types: [opened, reopened, edited, synchronize]
+    branches: ["*"]
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Cache node modules
+        uses: actions/cache@v2
+        env:
+          cache-name: cache-node-modules
+        with:
+          # npm cache files are stored in `~/.npm` on Linux/macOS
+          path: ~/.npm
+          key: ${{ runner.os }}-qa-${{ env.cache-name }}-${{ hashFiles('**/package-lock.json') }}
+          restore-keys: |
+            ${{ runner.os }}-qa-${{ env.cache-name }}-
+            ${{ runner.os }}-qa-
+            ${{ runner.os }}-
+
+      - name: Install deps
+        run: |
+          npm ci
+
+      - name: Set API URI
+        env:
+          pull_request_body: ${{ github.event.pull_request.body }}
+          prefix: API_URI=
+          pattern: (http|https)://[a-zA-Z0-9.]+/graphql/?
+        run: |
+          echo "::set-env name=API_URI::$(echo $pull_request_body | grep -Eo "$prefix$pattern" | sed s/$prefix// | head -n 1)"
```

Comment:
I don't like the name of the variable, we have such a variable name in PWA, the person could be mentioning it in the description. What about `DEPLOYMENT_API_URL=...` or "Target Deployment URL: ...`

Reply:
Again, env variable set in this step have to be the same as expected by storefront build script